app/models/email_history.rb

```diff
@@ -4,15 +4,22 @@ class EmailHistory < ApplicationRecord
   AUTO_EARLY_ALLOCATION = 'auto_early_allocation'
   DISCRETIONARY_EARLY_ALLOCATION = 'discretionary_early_allocation'
   SUITABLE_FOR_EARLY_ALLOCATION = 'suitable_for_early_allocation'
+  WELSH_TRANSFER_TO_PRESCOED = 'welsh_transfer_to_prescoed'
 
   validates_presence_of :name, :nomis_offender_id
 
   validates :event, inclusion: { in: [AUTO_EARLY_ALLOCATION,
                                       DISCRETIONARY_EARLY_ALLOCATION,
-                                      SUITABLE_FOR_EARLY_ALLOCATION], allow_nil: false }
+                                      SUITABLE_FOR_EARLY_ALLOCATION,
+                                      WELSH_TRANSFER_TO_PRESCOED], allow_nil: false }
   validates :prison, inclusion: { in: PrisonService.prison_codes, allow_nil: false }
   validates :email, presence: true, 'valid_email_2/email': true
 
+  # offender could have been at this prison multiple times, so order by newest first
+  scope :welsh_open_prescoed, lambda { |nomis_offender_id|
+    where(nomis_offender_id: nomis_offender_id, event: EmailHistory::WELSH_TRANSFER_TO_PRESCOED).order('created_at DESC')
+  }
```

Comment:
### Naming

Assuming the event name is changed to avoid being Prescoed-specific, I think the scope name should reflect the name of the event.

### Within offender sentence?

Is there also a need to check whether the email was sent within the offender's current sentence?

I remember we did this in the 'might be suitable for Early Allocation' email:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/883f6c38e04d20b508d61aeb7d2e97b2bb32821c/app/jobs/suitable_for_early_allocation_email_job.rb#L16-L18

In order to do that, we need to have the `Offender` object in our context. This would be possible of the helper method `welsh_offender_from_closed_to_prescoed?` was moved into the `Offender` object itself – you'd have the sentence start date directly in your hands.

And then since the query will only be used in one place, it might not actually be worth having this as a reuable scope. The query could live directly in the Offender method itself.